Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
-  (d) An organization may have no members as part. 
+    True. The organization United States Government has the Organization Department of Aggroculture as its part.
+  (d) An organization may have no members as part.
+    True. A club can lose its all members and regain some members in the next year. (of course one could argue whether it is the same club or not)
   (e) Any site is partially bounded by some instance of Material Entity.
-  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+    Flase. By the BFO defintion of Site, it must be wholy or partially bounded by a material entity or may be a three-dimensional immaterial part of an immaterial entity (partially or wholly) bounded by a material entity. 
+  (f) A book placed under the leg of a wobbly table has acquired a new function.
+    False. Unless the book was an artifact made solely for the purpose of being used as described, functions are realizable entities that an object is precisely made or evolved for that purpose.
   (g) A glass vase cushioned with packing material for all time, has the disposition to break. 
+    True. Disposition is relavent to the physicality of the object and not its environment.
   (h) Spacetime is a class in BFO.
+    False. I say false, just because it is technicaly Spatiotemporal Region that is a BFO class.
   (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+    True. This is with the assumption that we are talking about adding subclasses in BFO. It might be the case that a field-specific ontology would want to add subclasses.
 ```
 
 8. Model the following scenario in BFO, introducing whatever terms are needed to do so: John runs for 3 hours, startin slowly, speeding up during the middle, then ending the run at a slower pace.  
+![Project3Q8](https://user-images.githubusercontent.com/106703400/223338847-9105d7bd-d002-4367-8ab9-9fc2ea32f943.png)
+
+```
+(Comment: This is a graph based on Mermaid, representing the following relations:
+– John (as an Object) participates_in John’s running (as a process), and John bears John’s speeds 1, 2, and 3, all of which are instances of dispositions.
+–  John’s running includes 3 temporal parts: John’s beginning stage, John’s middle stage, and John’s final stage, each of which is a process. Besides, John’s beginning stage precedes John’s middle stage which precedes John’s final stage.
+– A process realizes one’s disposition. So John’s beginning stage realizes John’s speed 1, John’s middle stage realizes John’s speed 2, and John’s final stage realizes John’s speed 3.
+– Change in John’s speed: John's Speed 1 is decreased_in_magnitude_relative_to John's Speed 2 (that is, speed 1＜speed 2), and John's Speed 2 is increased_in_magnitude_relative_to John's Speed 3 (that is, speed 2＞speed 3)
+– John’s running occurs in this 3-hour, which is instance_of one-dimensional temporal region.)
+```
```

Comment:
According to Giacomo's and your nice catch, I adjust the picture by changing Speed as a relational quality. New picture can be found in our Google Doc File